Firestore/Source/Public/FIRFirestore.h

```diff
@@ -139,6 +139,24 @@ NS_SWIFT_NAME(Firestore)
 + (void)enableLogging:(BOOL)logging
     DEPRECATED_MSG_ATTRIBUTE("Use FIRSetLoggerLevel(FIRLoggerLevelDebug) to enable logging");
 
+
+#pragma mark - Network
+
+/**
+ * Re-enables usage of the network by this Firestore instance after a prior call to
+ * disableNetworkWithCompletion. Completion block, if provided, will be called once network uasge
```

Comment:
References to other code entities in comments should be in backticks.

Reply:
Done.